birdhouse/config/phoenix/config/canarie-api/canarie_api_monitoring.py.template

```diff
@@ -1,34 +0,0 @@
-SERVICES['Phoenix'] = {
```

Comment:
Forgot to move this into `deprecated-components`.

Reply:
https://github.com/bird-house/birdhouse-deploy/pull/311#discussion_r1218107135